internal/execution/executor.go

```diff
@@ -29,35 +29,42 @@ type Executor struct {
 
 func (e *Executor) Synthesize(ctx context.Context, env *Env) error {
 	logger := logr.FromContextOrDiscard(ctx)
+	logger.Info("starting synthesis", "synthesisuuid", env.SynthesisUUID)
 
 	comp := &apiv1.Composition{}
 	comp.Name = env.CompositionName
 	comp.Namespace = env.CompositionNamespace
 	err := e.Reader.Get(ctx, client.ObjectKeyFromObject(comp), comp)
 	if err != nil {
+		logger.Error(err, "unable to fetch composition")
```

Comment:
Is it possible to format the error logs to container which composition and which synthesizer?

Reply:
something like the below?
		logger.Error(err, fmt.Sprintf("failed to get composition. compition Name[%s] Namespace[%s]", comp.GetName(), comp.GetNamespace()))